jobs/pacman-rule-engine-2.0/src/main/java/com/tmobile/pacman/executor/PolicyExecutor.java

```diff
@@ -250,9 +250,9 @@ private void run(Map<String, String> policyParam, String executionId)
         policyParam.put(PacmanSdkConstants.Role_IDENTIFYING_STRING, PacmanSdkConstants.ROLE_PREFIX +
                 PacmanSdkConstants.INTEGRAION_ROLE);
         if (Strings.isNullOrEmpty(policyParam.get(PacmanSdkConstants.DATA_SOURCE_KEY))) {
-            logger.error(DATA_ALERT_ERROR_STRING + JOB_NAME + " with job id " + policyParam.get(PacmanSdkConstants.POLICY_UUID_KEY) +
-                    "data source is missing, will not be able to figure out the target index to post the policy evaluvation, please check policy configuration");
-            logger.error("exiting now..");
+            logger.error("data source is missing, will not be able to figure out the target index to post the policy " +

```

Comment:
simply this to:

data source missing in configuration, unable to post policy

Reply:
full message can be

"Exiting due to error -> data source missing in configuration, unable to post policy"

Reply:
updated